app/controllers/account_controller.rb

```diff
@@ -105,7 +105,7 @@ def signup # :nologin: :prefetch:
           flash_object_errors(@new_user)
         else
           group = UserGroup.create_user(@new_user)
-          flash_notice :runtime_signup_success.t
+          flash_notice (:runtime_signup_success.tp + :email_spam_notice.t)
```

Comment:
I'm concerned about mixing xxx.tp and xxx.t.  tp adds a <p> wrapper around the text.  If one needs it, don't both?  Same thing below.